language/move-prover/bytecode/src/spec_instrumentation.rs

```diff
@@ -392,6 +393,7 @@ impl<'a> Instrumenter<'a> {
     ) {
         use Bytecode::*;
         use PropKind::*;
+        let targs = &targs; // linter does not allow `ref targs` parameter
```

Comment:
There are some weird pass-by-ref vs pass-by-value thing here. My local experiment with removing this line and replacing all `targs` with `&targs` in this function solve the problem (with one exception: line 481, drop the `clone()`).

Reply:
But I don't want to use `&targs`, I want to use `targs` and be it a reference right away. On the other hand, I do not want to change the API of this function, because the caller can provide ownership. The `ref` modifier for a variable is usually the way to do this, just in this case, the linter asks me to change function signature...

Reply:
Here is the diff for the changes I mean, it does not change any function signature, just how the function arguments are passed.

[mono.diff.txt](https://github.com/diem/diem/files/6367228/mono.diff.txt)